velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -310,5 +313,38 @@ TEST_F(DateTimeFunctionsTest, dayOfMonth) {
   EXPECT_EQ(23, day(parseDate("2023-08-23")));
 }
 
+TEST_F(DateTimeFunctionsTest, addMonths) {
+  const auto parseDate =
+      [](const std::optional<const char*> date) -> std::optional<int32_t> {
+    if (date.has_value()) {
+      return DATE()->toDays(date.value());
+    }
+    return std::nullopt;
+  };
+
+  const auto addMonths =
+      [&](const std::optional<const char*> date,
+          std::optional<int32_t> value) -> std::optional<std::string> {
+    auto res = evaluateOnce<int32_t, int32_t>(
+        "add_months(c0, c1)", {parseDate(date), value}, {DATE(), INTEGER()});
+    if (res.has_value()) {
+      return DATE()->toString(res.value());
+    }
+    return std::nullopt;
+  };
+
+  EXPECT_EQ("2015-02-28", addMonths("2015-01-30", 1));
+  EXPECT_EQ("2015-01-30", addMonths("2015-01-30", 0));
+  EXPECT_EQ("2016-02-29", addMonths("2016-03-30", -1));
+  EXPECT_EQ("2014-11-30", addMonths("2015-01-30", -2));
+
+  constexpr int32_t kMin = std::numeric_limits<int32_t>::min();
+  constexpr int32_t kMax = std::numeric_limits<int32_t>::max();
+  VELOX_ASSERT_THROW(
+      addMonths("2023-07-10", kMin), "Integer overflow in add_months");
```

Comment:
Can you extend this test to verify the runtime information included in the error message? Let's make sure it includes both 2023-07-10 and kMin.